ern-orchestrator/src/runMiniApp.ts

```diff
@@ -161,14 +161,28 @@ export async function runMiniApp(
     outDir,
   })
 
+  const pathToRunner = path.join(cwd, platform)
+  const oldAndroidRunner = isOldAndroidRunner()
+
   if (platform === 'android') {
+    if (oldAndroidRunner) {
+      log.warn(
+        'Looks like you are running on an older version of the adroid runner project generated using ern version < 0.38.0. \n We recommend that you delete your current Android directory and regenerate the new runner by executing `ern run-android` command again.'
```

Comment:
typo `adroid` (should be `android`)

Reply:
Also - Related question (more general, not about this PR): When we suggest to "regenerate" the project - What happens if the user has made some modifications to the native files? Maybe we can discuss this separately at some point.. ;)

Reply:
Those changes will have to be reapplied. Perhaps We can have a mention of that here or altogether remove this warning.